proto/spire/api/types/selector.proto

```diff
@@ -20,6 +20,14 @@ message SelectorMatch {
         // Indicates that the selectors in this match are a subset of the
         // candidate selectors.
         MATCH_SUBSET = 1;
+
+        // Indicates that the selectors in this match are a superset of the
+        // candidate selectors.
```

Comment:
Hmm I think these comments imply the inverse of the comments in the FederatesWithMatch message. There, the term `candidate` is used for entries that will be returned by the query. Here, we are saying that MATCH_SUPERSET will return candidates that are a _subset_ of the provided selectors, and we say the opposite in FederatesWithMatch. FederatesWithMatch is the behavior we want.

Reply:
I apply some changes and add examples to all matches I think it can help us to avoid wording issues